src/index.ts

```diff
@@ -1 +1,22 @@
-console.log('hi');
+import express from 'express';
+import { userRouter } from './routers/user';
+import { ROUTERS_NAMES, PORT } from './configure/configure.constants';
+import { sequelize } from './models/index';
+
+const app = express();
+
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.get('/', (_req, res) => {
```

Comment:
This check isn't obligatory.

Reply:
I didn't completely undestand. Can you explain to me ?

Reply:
Done